lib/meadow_web/plugs/elasticsearch.ex

```diff
@@ -37,7 +37,13 @@ defmodule MeadowWeb.Plugs.Elasticsearch do
         Application.get_env(:meadow, Meadow.ElasticsearchCluster)
       )
 
-    case config |> config[:api].request(conn.method, path <> query_string, body, []) do
+    method = case conn.method do
```

Comment:
Is the accepted method check above on line 17 still working?

Reply:
Good catch. Probably not. I’ll normalize it Up front when I get back.

Reply:
Good catch. Probably not. I’ll normalize it Up front when I get back.

Reply:
I’ve untangled what Plug, Elasticsearch, and HTTPoison are doing with respect to HTTP methods, and I think it’ll simplify things to go with them.
`Plug.Conn`’s method attribute is always an uppercase binary (e.g., `"GET"`) (per [Plug.Conn docs](https://hexdocs.pm/plug/Plug.Conn.html))
* Both Elasticsearch APIs (`HTTP` and `AWS`) pass the method unaltered to `HTTPoison.request/5`, which expects an atom. (per [HTTPoison docs](https://hexdocs.pm/httpoison/HTTPoison.html#request/5))

So we should be fine assuming the method is an uppercase binary on the way in (for checking against `@accepted_methods`) and atomizing before sending it along to `api.request/5`. The truly maddening part is that `HTTPoison` just wants to turn it back into a binary. 🙄